catalyst/callbacks/soft_update.py

```diff
@@ -0,0 +1,47 @@
+from torch import nn
+
+from catalyst.core import CallbackOrder, IRunner
+from catalyst.core.callback import Callback
+
+
+def soft_update(target: nn.Module, source: nn.Module, tau: float) -> None:
+    """Updates the `target` data with the `source` one smoothing by ``tau`` (inplace operation)."""
+    for target_param, param in zip(target.parameters(), source.parameters()):
+        target_param.data.copy_(target_param.data * (1.0 - tau) + param.data * tau)
```

Comment:
let's move to `utils.torch`?

Reply:
Yeah, It makes sense.
[Done](https://github.com/catalyst-team/catalyst/pull/1305/commits/59875bc2ee0319f12d3acf4dfb5feb278c7358ef)